specs/toolsets/openxr_cppm/setup-toolset.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+export HAM_TOOLSET=openxr_cppm
+export HAM_TOOLSET_NAME=openxr_cppm
+export HAM_TOOLSET_DIR="${HAM_HOME}/specs/toolsets/openxr_cppm"
+
+BUILD=ra ham-cppm-build-check openxr_cppm noheader openxr_cppm openxr_cppm
+BUILD=da ham-cppm-build-check openxr_cppm noheader openxr_cppm openxr_cppm
+
+VER="--- openxr_cppm ---------------------"
```

Comment:
There's no executable or anything we can use to get a version? Not a big deal but nicer if available.

Reply:
mmm they run a cmake script to create the version...Maybe I could run that script only and get the result with cmake.

Reply:
No to cmake, that's a 300+ MB dependency.

Reply:
The Cmake script to create the version probably run a one line bash command and outputs it in a text file. Anyway its not really critical we can leave it for now.